PostHog/Replay/PostHogSessionReplayConfig.swift

```diff
@@ -24,10 +24,10 @@
         /// Default: true
         @objc public var maskAllSandboxedViews: Bool = true
 
-        /// Enable masking of images that likely originated from user's photo library
-        /// Experimental support (UIKit only)
+        /// Enable masking of images that may contain human faces
+        /// Experimental support
         /// Default: true
-        @objc public var maskPhotoLibraryImages: Bool = true
```

Comment:
We cannot do this otherwise we will break people's builds on react native and iOS.
If needed we would need a major bump in the sdk version.